.github/renovate.json

```diff
@@ -6,5 +6,5 @@
   "separateMajorMinor": false,
   "groupName": "all",
   "schedule": "before 6am every weekday",
-  "ignoreDeps": ["github-scm-filter-aged-refs", "azure-vm-agents"]
```

Comment:
bit worrying that this isn't maintained,

azure vm agents was supposed to be removed 7 months ago: https://github.com/hmcts/cnp-jenkins-docker/pull/1568 

It may be better to just set these to have automerge false temporarily instead of ignoring?